docs/installation/minikube-amd64.md

```diff
@@ -0,0 +1,192 @@
+# Kstone Installation
+
+[中文](README_CN.md)
```

Comment:
Link not found，it is better to provide a chinese installation document. If there is no time right now, we can remove this link first. thanks.